WD-Firmware/examples/build/SConscript_psp

```diff
@@ -15,34 +15,38 @@
  limitations under the License.
 '''
 import os
+import utils
 Import('Env')
 
 out_dir = os.path.join(Env['OUT_DIR_PATH'], 'psp')
-if not os.path.exists(out_dir):
-    os.makedirs(out_dir)
+utils.createFolder(out_dir)
 
 list_files_c=[
-   [os.path.join('psp','psp_interrupt.c'), os.path.join(out_dir, 'psp_interrupt.o')],
-   [os.path.join('psp', 'psp.c'), os.path.join(out_dir, 'psp.o')],
+   [os.path.join('psp', 'src', 'psp_interrupts.c'), os.path.join(out_dir, 'psp_interrupts.o')],
+   [os.path.join('psp', 'src', 'psp_traps.c'), os.path.join(out_dir, 'psp_traps.o')],
+   [os.path.join('psp', 'src', 'psp_timer.c'), os.path.join(out_dir, 'psp_timer.o')],
 ]
 
 list_files_a=[
-[os.path.join('psp', 'psp_int_vect.S'), os.path.join(out_dir, 'psp_int_vect.o')],
+   [os.path.join('psp', 'src', 'psp_int_vect.S'), os.path.join(out_dir, 'psp_int_vect.o')],
 ]
 
 # compiler flags
 c_flags = [] + Env['C_FLAGS']
 a_flags = [] + Env['A_FLAGS']
 
+
 # defines
-c_defines = []
+c_defines = []+Env['PUBLIC_DEF'] + Env['RTOSAL_DEF']
 
 # include paths
 local_includes  = []
 public_includes = [
   os.path.join(Env['ROOT_DIR'], 'psp', 'api_inc'),
-  os.path.join(Env['ROOT_DIR'], 'common', 'api_inc'),
+  os.path.join(Env['ROOT_DIR'], 'psp', 'psp_specific', 'loc_inc'),

```

Comment:
take to public only public

Reply:
clean this all public. it should only get Env["PUBLIC_INC"]

Reply:
done